06_gpu_and_ml/stable_diffusion_cli.py

```diff
@@ -170,18 +170,17 @@ def entrypoint(prompt: str, samples: int = 5, steps: int = 10, batch_size: int =
     if not dir.exists():
         dir.mkdir(exist_ok=True, parents=True)
 
-    with stub.run():
```

Comment:
Oops, my bad, missed this when fixing the examples. Interesting that it still worked 😬